orttraining/orttraining/core/session/tensor_helper.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/framework/ml_value.h"
+#include "core/session/inference_session.h"
+
+namespace onnxruntime {
+namespace training {
+OrtValue SliceTensor(const OrtValue& orig_value, const size_t slice_id,
+                     const size_t slice_axis, const size_t num_slices, onnxruntime::InferenceSession& session_state);
+}  // namespace training
+}  // namespace onnxruntime
```

Comment:
Not sure if this is the right place to put this function -- given it is pipeline specific function, maybe we can consider move it to the same folder as pipeline.h?

Reply:
Sure. I will move it.

---
In reply to: [538730261](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r538730261) [](ancestors = 538730261)